### DISCUSSION TITLE
Add Rallly to the Booking and Scheduling section of the awesome-selfhosted GitHub repository

### DISCUSSION THREAD

**Discussion by iamericfletcher** [lukevella/rallly#868] [General]
Came across this popular repository: [awesome-selfhosted](https://github.com/awesome-selfhosted/awesome-selfhosted)

Thought it might be beneficial to add Rallly to the Booking and Scheduling section.

What do you think?

**Comment by lukevella** [comment]
Hey Eric, you're right that would be a great spot for it. It's already in the list but it's under "Communication - Custom Communication Systems". I'll ask to have it moved. Thanks for pointing it out!

**Comment by iamericfletcher** [comment]
Excellent :) Sounds good!
